src/libraries/System.IO.Hashing/tests/XxHash64Tests.007.cs

```diff
@@ -114,6 +114,31 @@ public static IEnumerable<object[]> TestCases
         protected override bool TryStaticOneShot(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesWritten) =>
             XxHash64.TryHash(source, destination, out bytesWritten, Seed);
 
+        public static IEnumerable<object[]> LargeTestCases
+        {
+            get
+            {
+                object[] arr = new object[1];
+
+                foreach (LargeTestCase testCase in LargeTestCaseDefinitions)
+                {
+                    arr[0] = testCase;
+                    yield return arr;
+                }
+            }
+        }
+
+        protected static IEnumerable<LargeTestCase> LargeTestCaseDefinitions { get; } =
+            new[]
+            {
+                //https://asecuritysite.com/encryption/xxHash, Example 1
```

Comment:
I see example 1 as

`Try "Nobody inspects the spammish repetition". Try!, which should give a hash of: 0xe2293b2f`